packages/@dcl/inspector/src/components/Block/types.ts

```diff
@@ -1,3 +1,4 @@
 export type Props = {
   label?: string
+  broken?: boolean
```

Comment:
usually for these kind of properties we use the name `error` instead of broken, what do you think?